tutorial/debezium-cassandra-init/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM cassandra:3.11
+
+RUN apt-get update && \
+    apt-get install -y curl sudo  && \
+    sudo apt install -y openjdk-8-jdk
+
+ENV DEBEZIUM_VERSION=1.4.0.Alpha2 \
```

Comment:
Let's use 1.3.1.Final then, so to be in sync with 1.3 above.

Reply:
yes @gunnarmorling! Updated.